inst/documents/dummy.qmd

```diff
@@ -62,9 +62,10 @@ knitr::spin_child(params$script)
 #| include: false
 
 python_packages <- NULL
-# if (file.exists(file.path(params$tmpdir, "python_imports.json"))){
-#   python_packages <- whirl:::python_package_info(file.path(params$tmpdir, "python_imports.json"))
-# }
+
+if (file.exists(file.path(params$tmpdir, "python_imports.json"))){
+  python_packages <- whirl:::python_package_info(file.path(params$tmpdir, "python_imports.json"))
```

Comment:
because dummy.qmd is accessible to the user (inside inst), may be we should export the python package function, to avoid side effect one day

Reply:
It is a bit tricky that we are calling the internal functions from the Quarto documents, but I would not like to export them since they are not meant to be used by the users.
We have a handful of functions that are being called like this.